app/_kong_plugins/rate-limiting-advanced/index.md

```diff
@@ -139,3 +178,19 @@ You can use the [Consumer Groups entity](/gateway/entities/consumer-group/) to m
 subsets of Consumers.
 
 You can see an example of this in the guide on [enforcing rate limiting tiers with the Rate Limiting Advanced plugin](/how-to/add-rate-limiting-tiers-with-kong-gateway/).
+
+## Throttle rate limits {% new_in 3.12 %}
+
+In {{site.base_gateway}} 3.12, you can enable request throttling via the Rate Limiting Advanced plugin to improve clients' experience and protect upstream origin servers from being overwhelmed by traffic spikes. By throttling requests, this allows requests that exceed the rate limit threshold to be delayed and retried, rather than immediately rejected with a `429` status code. 
```

Comment:
```suggestion
In {{site.base_gateway}} 3.12 or later, you can enable request throttling using the Rate Limiting Advanced plugin to improve clients' experience and protect upstream origin servers from being overwhelmed by traffic spikes. With throttling, requests that exceed the rate limit threshold can be delayed and retried, rather than immediately rejected with a `429` status code. 
```